server/tests/util/db.js

```diff
@@ -14,6 +15,7 @@ async function clearDB() {
 async function startDB() {
   await dbClient.connect();
   await clearDB();
+  await keycloak.buildInternalIdMap();
```

Comment:
It's not in your changes but just curiosity. Why are there two lines of authenticateIfNeed in buildInternalIdMap method?

Reply:
KeyCloak Manager used a service account to interact with KeyCloak server. 
`authenticateIfNeed`  => 
    1. Check expiry of auth token 
    2. Renew auth token if expired.